src/transformers/models/jukebox/tokenization_jukebox.py

```diff
@@ -0,0 +1,424 @@
+# coding=utf-8
+# Copyright 2022 The Open AI Team Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for OpenAI Jukebox."""
+
+
+import json
+import os
+from json.encoder import INFINITY
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+
+import regex as re
+from tokenizers import normalizers
```

Comment:
Hmm this mixes a bit fast and slow tokenizers, e.g. one has to have `tokenizers` installed to now use this tokenizer. 
Could we maybe also just use Python functionality instead of the fast tokenizers here? 

What do you think @sgugger ?

Reply:
Yes, slow tokenizers should not depend on the tokenizers library at all.

Reply:
Will use python functions to replace that, good catch